tests/sandbox.py

```diff
@@ -118,8 +130,11 @@ def add_mrjob_to_pythonpath(self):
         (Merely using the local runner won't require this, because it
         bootstraps mrjob by default.)
         """
-        mrjob_pythonpath = os.path.abspath(
-            os.path.join(os.path.dirname(mrjob.__file__), '..'))
-
         os.environ['PYTHONPATH'] = (
-            mrjob_pythonpath + ':' + os.environ.get('PYTHONPATH', ''))
+            mrjob_pythonpath() + ':' + os.environ.get('PYTHONPATH', ''))
+
+
+def mrjob_pythonpath():
+    """The directory containing the mrjob package that we've imported."""
+    return os.path.abspath(
+        os.path.join(os.path.dirname(mrjob.__file__), '..'))
```

Comment:
That doesn't make sense, to me. These things are strictly string munging, with no filesystem access. In this example, bar  is neither a file nor a directory:

```python
>>> os.path.abspath(os.path.join('/foo/bar', '..'))
'/foo'
```

Anyhow, not important.